GitUI/Editor/FileViewer.cs

```diff
@@ -234,8 +182,6 @@ public new Font Font
             set => internalFileViewer.Font = value;
         }
 
-        public Action OpenWithDifftool => internalFileViewer.OpenWithDifftool;
```

Comment:
Not opposing removal, just an observation: I kept intentionally after #7891 as the handling in FileViewerInternal should maybe be adapted instead, or removed too. With the follow up to #7899 and #7825 it makes sense to keep this in FileViewer instead of spreading in other forms.

Reply:
Can we add it when it is required? At the moment it just add noise.

Reply:
Of course, just noting the motivation to why not removing earlier and adding another note about the intenal opendifftool